nautobot/core/api/views.py

```diff
@@ -198,7 +198,7 @@ def get_serializer(self, *args, **kwargs):
         return super().get_serializer(*args, **kwargs)
 
     def get_serializer_class(self):
-        logger = logging.getLogger("nautobot.core.api.views.ModelViewSet")
+        logger = logging.getLogger(__name__ + ".ModelViewSet")
```

Comment:
Not blocking for this PR but it'd probably be reasonable to make this a class attribute so that we don't have to look it up over again each time here and below.